build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/info/GlobalBuildInfoPlugin.java

```diff
@@ -113,7 +125,9 @@ public void apply(Project project) {
             params.setBuildDate(ZonedDateTime.now(ZoneOffset.UTC));
             params.setTestSeed(getTestSeed());
             params.setIsCi(System.getenv("JENKINS_URL") != null || System.getenv("BUILDKITE_BUILD_URL") != null);
-            params.setDefaultParallel(ParallelDetector.findDefaultParallel(project));
+            // params.setDefaultParallel(ParallelDetector.findDefaultParallel(project));
```

Comment:
Didn't we resolve the configuration cache issue here in a separate PR?

Reply:
yes. removed that bit